src/views/search/search-conversation-list-item.tsx

```diff
@@ -149,13 +202,13 @@ const SearchListItem = ({ itemId, item, folderId, selected, selecting, toggle, a
 				>
 					<Container orientation="horizontal" height="fit" width="fill">
 						<SenderName item={item} isFromSearch />
-						<RowInfo item={item} tags={tags} />
+						<RowInfo item={item} tags={tags} isFromSearch allMessagesInTrash={allMessagesInTrash} />
 					</Container>
 					<Container orientation="horizontal" height="fit" width="fill" crossAlignment="center">
-						{item.msgCount > 1 && (
+						{item.messages.length > 1 && (
 							<Row>
 								<Padding right="extrasmall">
-									<Badge value={item.msgCount} type={item.read ? 'read' : 'unread'} />
+									<Badge value={msgToDisplayCount} type={item.read ? 'read' : 'unread'} />
```

Comment:
if we have msgToDisplayCount then I guess you should use this in line no. 208 for checking the rendering condition